packages/simcore-sdk/tests/unit/test_node_ports_v2_port.py

```diff
@@ -77,18 +78,51 @@ def e_tag() -> str:
 ##################### FIXTURES
 
 
-@pytest.fixture
-def this_node_file(tmp_path: Path) -> Path:
-    file_path = this_node_file_name()
-    file_path.write_text("some dummy data")
+@pytest.fixture(
+    params=[
+        True,
+        False,
+    ]
+)
+def is_symlink(request) -> bool:
```

Comment:
this fixture name is pretty unclear. are we allowing for symlink? or are we creating a symlink?

Reply:
`return_symlink` seems to be a more adequate name